extensions/rates/PREINSTALL.md

```diff
@@ -0,0 +1,61 @@
+Use this extension to fetch rates for shipments from documents added to a specified Cloud Firestore collection.
```

Comment:
Use this extension to calculate real-time shipping rates across 50+ global carriers from documents added to a specified Cloud Firestore collection. [ShipEngine](https://www.shipengine.com/) retrieves all possible rates to help brands, ecommerce platforms, and 3PLs to make the best choice for every shipment, whether shipping cost, time to delivery, carrier capabilities or other factors are most important.